core/src/Threads/Kokkos_Threads_Team.hpp

```diff
@@ -1001,8 +1001,10 @@ KOKKOS_INLINE_FUNCTION void parallel_scan(
     lambda(i, scan_val, false);
   }
 
+  auto team_member = loop_bounds.thread;
```

Comment:
```suggestion
  auto& team_member = loop_bounds.thread;
```
so we would not accidentally update a copy.